packages/cli/src/utils.ts

```diff
@@ -185,6 +363,32 @@ export const validFields = {
       forceStrategy
     );
   },
+  exports(pkg: Package, forceStrategy?: DistFilenameStrategy) {
+    if (typeof pkg.json.exports === "undefined") {
+      return;
+    }
+    const conditions = Object.values(pkg.json.exports);
```

Comment:
Those could be direct mappings~ and not conditions - like for example when we are dealing with `package.json`. So perhaps renaming this variable would be preferred here.

Reply:
Additionally, we can't assume that `pkg.json.exports` is an object here so it probably would be good to be defensive about it.

Reply:
We're no longer checking `package.json.exports` to determine what conditions you would like to opt into and instead we're reading from the `exports` field in the preconstruct config (if defined).